### PR TITLE
EN-66163: Added jakarta so that we can support Java 9+

### PR DESCRIPTION
**Pull request by PiSoup**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,19 +4,19 @@ name := "eurybates"
 
 scalaVersion := "2.12.8"
 
-crossScalaVersions := Seq(scalaVersion.value)
```
Comment (PiSoup) [inline build.sbt:7]: Not much point in this.

File: .tool-versions [hunk 2]
```diff
@@ -1 +1 @@
-sbt 1.3.13
+sbt 1.5.8
```
Comment (PiSoup) [inline .tool-versions:1]: So it agrees with the properties.

File: build.sbt [hunk 3]
```diff
@@ -4,19 +4,19 @@ name := "eurybates"
...
-
 resolvers += "socrata" at "https://repo.socrata.com/artifactory/libs-release"
 
 mimaPreviousArtifacts := Set("com.socrata" %% "eurybates" % "2.1.0")
 
 libraryDependencies ++= Seq(
   "com.rojoma" %% "rojoma-json-v3" % "3.10.0",
   "com.socrata" %% "socrata-zookeeper" % "1.1.0",
-  "org.apache.activemq" % "activemq-core" % "5.7.0" % "optional",
```
Comment (PiSoup) [inline build.sbt:16]: 5.7.0 has an astonishing number of CVEs anyways.

File: project/plugins.sbt [hunk 4]
```diff
@@ -3,4 +3,3 @@ resolvers ++= Seq(
 )
 
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.6.1")
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```
Comment (PiSoup) [inline project/plugins.sbt:6]: `dependencyTree` is just part of sbt these days.

File: src/main/scala/com/socrata/eurybates/Message.scala [hunk 5]
```diff
@@ -3,4 +3,4 @@ package eurybates
 
 import com.rojoma.json.v3.ast.JValue
 
-case class Message(tag: Tag, details: JValue, activities: JValue)
```
Comment (PiSoup) [inline src/main/scala/com/socrata/eurybates/Message.scala:6]: Walking back the incomplete changes from the latest commit.

File: src/main/scala/com/socrata/eurybates/MessageCodec.scala
Comment (PiSoup) [inline src/main/scala/com/socrata/eurybates/MessageCodec.scala:1]: These are all incomplete changes from the previous commit which do not work.

**Review by ethorne — APPROVED** [review]

**Pull request merged**